### PR TITLE
Added guide for context-menu

### PR DESCRIPTION
**Pull request by ganeshpatil386386**
Created a completed guide for context-aware-menus with well-explained and covered all the options including 
- Set as variable
- Add to parameters 
- Open req in a new tab

Need reviews - @SanskritiHarmukh @thetronjohnson

### CODE REVIEW

File: documentation/features/context-menu.mdx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+Context-aware menus enable quick actions like creating variables or adding query
+parameters by selecting text or elements. The context menu offers three options:
+**Set as Variable**, **Add to Parameters**, and **Open Request in New Tab**.
+
+<Info>
+  The "Add to Parameters" option is available only when using query parameters.
+</Info>
+
+## Set as Variable
```
Comment (SanskritiHarmukh) [inline documentation/features/context-menu.mdx:15]: Just paraphrasing this section to add more clarity while reading:

> You can use the **Set as Variable** option from the context menu to assign a selected value to an environment variable.
> 
> Hoppscotch supports various variable scopes, including global, environment, and request-specific variables. For more details on these scopes, [click here](/documentation/features/variables).

can you make a new commit for this?

File: documentation/features/context-menu.mdx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+You can access the **Set as Variable** option from the context menu. Once
+selected, you can assign the selected value to an environment variable.
+
+Hoppscotch offers different variable scopes, including global, environment, and
+request variables. You can read more about variable scopes
+[here](/documentation/features/variables).
+
+### How to Use "Set as Variable"
```
Comment (SanskritiHarmukh) [inline documentation/features/context-menu.mdx:24]:
```suggestion
### How to use "Set as Variable"
```

File: documentation/features/context-menu.mdx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+After setting an [environment variable](/documentation/features/environments),
+you can use it with the format `<<variable name>>`. This allows you to store
```
Comment (SanskritiHarmukh) [inline documentation/features/context-menu.mdx:27]:
```suggestion
you can reference it using the format `<<variable name>>`. 
```

File: documentation/features/context-menu.mdx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+values like `<<baseURL>>` in a single variable.
```
Comment (SanskritiHarmukh) [inline documentation/features/context-menu.mdx:28]:
```suggestion
For example, `<<baseURL>>` can be used to dynamically insert the base URL across your requests.
```

File: documentation/features/context-menu.mdx [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+## Add to Parameters
+
+The **Add to Parameters** option lets you easily add parameters and their
```
Comment (SanskritiHarmukh) [inline documentation/features/context-menu.mdx:32]:
```suggestion
The **Add to Parameters** option allows you to quickly add parameters along with their
```

File: documentation/features/context-menu.mdx [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+values.
+
+When you select **Add to Parameters**, it automatically captures the parameter
+and its value. For example, the `baseURL` is
```
Comment (SanskritiHarmukh) [inline documentation/features/context-menu.mdx:36]:
```suggestion
and its value. For example, if the `baseURL` is
```

File: documentation/features/context-menu.mdx [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+`https://jsonplaceholder.typicode.com/?user=1` now the user with ID "1" will be
```
Comment (SanskritiHarmukh) [inline documentation/features/context-menu.mdx:37]:
```suggestion
`https://jsonplaceholder.typicode.com/?user=1`, now the **user** with ID **1** will be
```

File: documentation/features/context-menu.mdx [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+included in the parameter placeholder.
+
+## Open Request in New Tab
+
+The **Open Request in New Tab** option allows you to open the current request in
```
Comment (SanskritiHarmukh) [inline documentation/features/context-menu.mdx:42]:
```suggestion
The **Open Request in New Tab** option allows you to open any **selected URL or text** as
```

File: documentation/features/context-menu.mdx [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+a new tab, enabling you to handle multiple requests simultaneously.
```
Comment (SanskritiHarmukh) [inline documentation/features/context-menu.mdx:43]:
```suggestion
a new request in a separate tab.
```

File: documentation/features/context-menu.mdx [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+
+This feature helps you shift the current request to a new tab while working on a
```
Comment (SanskritiHarmukh) [inline documentation/features/context-menu.mdx:45]:
```suggestion
By opening the selected URL in a new tab, you can continue working on your original request in its own tab without any interruptions.
```

File: documentation/features/context-menu.mdx [hunk 11]
```diff
@@ -0,0 +1,46 @@
...
+new request in the original tab.
```
Comment (SanskritiHarmukh) [inline documentation/features/context-menu.mdx:46]:
```suggestion

```

File: documentation/features/context-menu.mdx [hunk 12]
```diff
@@ -0,0 +1,48 @@
...
+
+Context-aware menus enable quick actions like creating variables or adding query
+parameters by selecting text or elements. The context menu offers three options:
+**Set as Variable**, **Add to Parameters**, and **Open Request in New Tab**.
+
+<Info>
+  The "Add to Parameters" option is available only when using query parameters.
+</Info>
+
+## Set as Variable
```
Comment (SanskritiHarmukh) [inline documentation/features/context-menu.mdx:16]: Hey, thanks for making the changes. Could you please remove the quotes (>) from this section? They were only for your reference.

File: documentation/features/context-menu.mdx [hunk 13]
```diff
@@ -0,0 +1,48 @@
...
+across your requests.
+
+## Add to Parameters
+
+The **Add to Parameters** option allows you to quickly add parameters along with
+their values.
+
+When you select **Add to Parameters**, it automatically captures the parameter
+and its value. For example, if the `baseURL` is
+`https://jsonplaceholder.typicode.com/?user=1`, now the **user** with ID **1**
```
Comment (SanskritiHarmukh) [inline documentation/features/context-menu.mdx:39]:
```suggestion
`https://jsonplaceholder.typicode.com/?user=1`, now the `user` with ID `1`
```

**Review by SanskritiHarmukh — CHANGES_REQUESTED** [review]

File: documentation/features/context-menu.mdx (hunk 13 above)
Comment (SanskritiHarmukh) [reply documentation/features/context-menu.mdx:39]: Can you tag me once these changes are done?

Comment (ganeshpatil386386) [reply documentation/features/context-menu.mdx:39]: Sure!

Comment (ganeshpatil386386) [reply documentation/features/context-menu.mdx]: @SanskritiHarmukh review changes!

**Review by SanskritiHarmukh — APPROVED** [review]

**Comment by SanskritiHarmukh** [comment]
LGTM ✅. @thetronjohnson , could you please take a quick look at this PR to see if anything else needs to be changed?

**Review by thetronjohnson — APPROVED** [review]

**Pull request merged**